frontend/src/components/Contact/Contact.jsx

```diff
@@ -1,25 +1,34 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { useLocation, Link } from 'react-router-dom';
 import { Button, Select, TextInput, RadioGroup, Radio } from '@dataesr/react-dsfr';
 
 import Page from 'components/Page/Page';
 import GlobalNotification from 'components/Notification/GlobalNotification';
 import agent from 'services/agent';
 import { useStore } from 'stores/';
-import { Link } from 'react-router-dom';
 
 const Contact = () => {
-  const [user, setUser] = useState('');
-  const [name, setName] = useState('');
-  const [firstName, setFirstName] = useState('');
-  const [email, setEmail] = useState('');
-  const [reason, setReason] = useState('');
-  const [message, setMessage] = useState('');
+  const [userType, setUserType] = useState();
+  const [name, setName] = useState();
+  const [firstName, setFirstName] = useState();
+  const [email, setEmail] = useState();
+  const [reason, setReason] = useState();
+  const [message, setMessage] = useState();
 
-  const { commonStore: { setNotification, config } } = useStore();
+  const { commonStore: { setNotification, config }, userStore: { user } } = useStore();
```

Comment:
I didn't use `observer` here on purpose despite the user store usage. It's working without it.

Reply:
Adding it anyway